www/_posts/2024-04-03-cordova-ios-7.1.0.md

```diff
@@ -0,0 +1,104 @@
+
+* [cordova-ios@7.1.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@7.1.0
+```
+
+**To install:**
+
+```bash
+cordova platform add ios@7.1.0
+```
+
+**Features:**
+
+* **`SplashScreenBackgroundColor` preference support**
+
+    This preference allows you to set the splashscreen's background colour. If no `SplashScreenBackgroundColor` is provided, it will fall back to the `BackgroundColor`, which is the current behavior, and then fallback to the system background colour when nothing is defined.
+
+* **`privacy-manifest` Supprot**
+
+    This release supplies the templated blank privacy manifest file, `PrivacyInfo.xcprivacy` which can be configured from `config.xml`.
+
+    This release focuses specifically on providing support for app developers to set this configuration, while a later release will introduce support for plugin developers.
+
+    It is recommended that plugin developers help app developers by providing in their plugin documentation the necessary configuration setup.
+
+    Below is an example config.xml entry that app developers can define to configure the privacy manifest file.
+
+    ```xml
+    <platform name="ios">
+        <privacy-manifest>
+            <key>NSPrivacyTracking</key>
+            <true/>
+            <key>NSPrivacyCollectedDataTypes</key>
+            <array/>
+            <key>NSPrivacyAccessedAPITypes</key>
+            <array/>
+            <key>NSPrivacyTrackingDomains</key>
+            <array/>
+        </privacy-manifest>
+    </platform>
+    ```
+
+    The contents and values of `NSPrivacyTracking`, `NSPrivacyCollectedDataTypes`, `NSPrivacyAccessedAPITypes`, and `NSPrivacyTrackingDomains` will depend on the specific native APIs being utilized.
+
+    It is recommended to read the Apple Developer documentation to better understand these parameters.
+
+    * [Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files)
+    * [Describing use of required reason API](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api)
+    * [Describing data use in privacy manifests](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests)
+
+**Fixes:**
+
+* **Use `PROVISIONING_PROFILE_SPECIFIER` for manual codesigning**
+
+    Previosuly, the `PROVISIONING_PROFILE` flag was being set for manual codesigning. This flag only accepted the provisioning profile's `UUID`. To accept the provisioning profile's `name` value, the `PROVISIONING_PROFILE_SPECIFIER` flag should be set instead. The `PROVISIONING_PROFILE_SPECIFIER` flag accepts both `name` and `UUID` values for the specified provisioning profile.
```

Comment:
typo: Previously

Reply:
This should be reworded to describe the impact to end users rather than the technical details. IIRC this means you can now specify a provisioning profile name in build.json or on the command line?